modules/core/src/main/scala/gem/config/GcalConfig.scala

```diff
@@ -1,14 +1,16 @@
 package gem
 package config
 
-import gem.enum.{GcalArc, GcalContinuum, GcalShutter}
+import gem.enum.{GcalArc, GcalContinuum, GcalDiffuser, GcalFilter, GcalShutter}
+
+import java.time.Duration
 
 import scalaz._
 import Scalaz._
 
 import GcalConfig.GcalLamp
 
-case class GcalConfig(lamp: GcalLamp, shutter: GcalShutter) {
+case class GcalConfig(lamp: GcalLamp, filter: GcalFilter, diffuser: GcalDiffuser, shutter: GcalShutter, exposureTime: Duration, coadds: Int) {
```

Comment:
Good question, I think I may be repeating a flaw of the current program model.  In today's Science Program model you can configure the instrument exposure time separately from the calibration step exposure time.  You then actually get three properties in the sequence step:

1. `calibration:exposureTime`
2. `instrument:exposureTime`
3. `observe:exposureTime`

The calibration and instrument exposure times may differ but the `observe:exposureTime` always matches one or the other depending on step type.  So I think this may make more sense as part of the step itself.  It is never absent, though it can be 0 seconds.

The one mitigating factor is thinking ahead to smart gcal.  There we'll want to associate gcal step configurations with subsets of instrument configurations and I guess it would be convenient to have the entire thing in `GcalConfig`.  I suppose we could always pair `GcalConfig` with the exposure time.

Coadds are very similar.  Not all instruments (and therefore not all steps) support coadds though.

If it seems okay, I'll merge this as is and we can rework it as necessary?